sunpy/util/util.py

```diff
@@ -195,3 +197,54 @@ def hash_file(path):
             sha256.update(data)
 
     return sha256.hexdigest()
+
+
+def get_rectangle_coordinates(bottom_left, *, top_right = None, width: u.deg = None, height: u.deg = None):
+    """
+    Returns bottom_left and top_right coordinates of a 2D rectangular region of interest.
+
+    Parameters
+    ----------
+        bottom_left : `astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`
```

Comment:
iirc there shouldn't be this indent after the heading, so all these lines should be dedented 4 spaces.